CONTRIBUTING.md

```diff
@@ -2,6 +2,6 @@
 
 Do you want to hack on Pulumi?  Awesome!  We are so happy to have you.
 
-Please refer to the [main Pulumi repo](https://github.com/pulumi/home/)'s [CONTRIBUTING.md file](
-https://github.com/pulumi/home/blob/master/CONTRIBUTING.md) for details on how to do so.
+Please refer to the [docs Pulumi repo](https://github.com/pulumi/docs/)'s [CONTRIBUTING.md file](
+https://github.com/pulumi/docs/blob/master/CONTRIBUTING.md) for details on how to do so.
```

Comment:
@ellismg Do we need to fix this up more broadly?

Reply:
I think this is still needed - I'll cherry pick it on top of the latest bits.